mypy.ini

```diff
@@ -1,5 +1,28 @@
 [mypy]
-ignore_missing_imports = True
-# required to support namespace packages
-# https://github.com/python/mypy/issues/14057
+# Is the project well-typed?
+strict = False
+
+# Early opt-in even when strict = False
+warn_unused_ignores = True
+warn_redundant_casts = True
+enable_error_code = ignore-without-code
+
+# Support namespace packages per https://github.com/python/mypy/issues/14057
 explicit_package_bases = True
+
+# Disable overload-overlap due to many false-positives
+disable_error_code = overload-overlap
+
+exclude = (?x)(
+    # upstream
+	^build/
+	| ^.tox/
+ 
+ 	# local
+	)
+
+# - most jaraco packages aren't marked as typed
+# - wheel is not marked as typed
+[mypy-jaraco.*,wheel.*]
+ignore_missing_imports = True
```

Comment:
I've decided to remove this for now. If a project needs to be excluded, we can exclude it, but let's not add everything in the jaraco namespace, which is dozens of projects. Any exclusion should have a comment linking to the upstream issue that will fix the exclusion.